README.md

```diff
@@ -3,7 +3,9 @@
 ## Introduction
 
 This repository is providing several tools needed to build LXQt itself as well
-as other components maintained by the LXQt project.
+as other components maintained by the LXQt project. 
+
+In LXQt 2.0.0 and later `lxqt2-` prefix is used for building with Qt6.
```

Comment:
This can be confusing because compilation itself is like before and doesn't need an extra prefix.

Package maintainers may choose different ways of supporting parallel installations in their distros.